extensions/wikia/MyHome/data/DataFeedProvider.php

```diff
@@ -230,7 +229,7 @@ private function filterEdit( DataFeedProviderItemResult $itemResult, $res, Title
 				if ( isset( $res['rc_params']['summary'] ) ) {
 					$item['comment'] = $res['rc_params']['summary'];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "comment" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)